src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
```

Comment:
Consider storing repeated values such as the size of the stars in SCSS variables for better maintainability and to adhere to DRY (Don't Repeat Yourself) principle. You can define these variables in `src/styles/utils/variables.scss`.